recipes/sparsestack/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License

```

Comment:
Why do you package a license file? It seems that the license file is already in the tarball?

Reply:
That's correct. I will hence remove it from here.

Reply:
@saraedum this should be resolved. Let me know if any other changes are needed.